src/lib.rs

```diff
@@ -56,8 +57,6 @@ use std::fmt;
 use std::error;
 use libc::PATH_MAX;
 
-pub type Result<T> = result::Result<T, Error>;
```

Comment:
Yeah, I'll do it when I get a chance sometime tomorrow night. Don't let that keep you from reviewing the rest of it though!